src/js/__tests__/classes/ArrayValidator.test.ts

```diff
@@ -126,3 +127,15 @@ test('Test ArrayValidator does not allow values that are not arrays', () => {
 
     expect(validator.validate('string' as any)).toStrictEqual([false, 'That is not an array :o']);
 });
+
+test('Test ArrayValidator has a default return value for non array values', () => {
```

Comment:
tests in this file are always calling `test` instead of `it` - is there any particular reason for this? Other test files seem to use `it`.